dask/array/creation.py

```diff
@@ -683,7 +684,10 @@ def _diag_len(dim1, dim2, offset):
 
 
 @derived_from(np)
-def tri(N, M=None, k=0, dtype=float, chunks="auto", like=None):
+def tri(N, M=None, k=0, dtype=float, chunks="auto", *, like=None):
+    if not _numpy_120:
```

Comment:
I think we want a `like is not None` here too

Reply:
Good catch, done in https://github.com/dask/dask/pull/8054/commits/37c85ad08dc9e4bc068f75ef837d5eadc6bbafce .